src/nfs-ganesha/main.c

```diff
@@ -60,7 +60,7 @@ struct SaunaFSModule SaunaFS = {
                  .acl_support = 0,
 #endif
                  .cansettime = true,
-                 .homogenous = true,
+                 .homogenous = true,  // pragma: codespell-ignore
```

Comment:
Thank you very much @yarikoptic for your contribution.
In this case, `homogenous` is the name of an attribute in `struct fsal_staticfsinfo_t`, which is a Ganesha definition. So, even if it there is a typo in the name, we should use it like it is to avoid compilation errors.

Being said that, I think adding the rule to ignore the typo is the correct solution :+1: